datafusion/core/tests/sql/avro.rs

```diff
@@ -121,11 +121,8 @@ async fn avro_single_nan_schema() {
     .await
     .unwrap();
     let sql = "SELECT mycol FROM single_nan";
-    let plan = ctx.create_logical_plan(sql).unwrap();
```

Comment:
🙈  -- this pattern is very old in the DataFusion codebase -- I think it was simply copied around and has never been cleaned up. Thank you 🙏